<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string notr="true">Пароль</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="passwordEdit">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string notr="true">Пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Опции</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="upperCaseCheck">
         <property name="text">
          <string notr="true">Большие буквы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lowerCaseCheck">
         <property name="text">
          <string notr="true">Маленькие буквы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="digitsCheck">
         <property name="text">
          <string notr="true">Цифры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="symbolsCheck">
         <property name="text">
          <string notr="true">Символы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string notr="true">Длина:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lengthComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string notr="true">Файл</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVariants">
    <property name="title">
     <string notr="true">Варианты</string>
    </property>
    <addaction name="actionMedium_Password"/>
    <addaction name="actionLong_Password"/>
    <addaction name="separator"/>
    <addaction name="actionUNIX"/>
    <addaction name="actionPIN"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string notr="true">Правка</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVariants"/>
  </widget>
  <action name="actionGenerate">
   <property name="text">
    <string notr="true">Генерировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string notr="true">Выход</string>
   </property>
  </action>
  <action name="actionMedium_Password">
   <property name="text">
    <string notr="true">Средний</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLong_Password">
   <property name="text">
    <string notr="true">Длинный</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionUNIX">
   <property name="text">
    <string>UNIX</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionPIN">
   <property name="text">
    <string>PIN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string notr="true">Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
